Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,144 @@
+        private string operand2;
+        private string expectedOperand2;
+        private string operation;
+        private string result;
+        private string expectedResult;
+        private void setOperand1(string value){
+            operand1=value;
+        }
+        public string getOperand1(){
+            return operand1;
+        }
+        public string getExpectedOperand1(){
+            return expectedOperand1;
+        }
+        private void setOperand2(string value){
+            operand2=value;
+        }
+        public string getOperand2(){
+            return operand2;
+        }
+        public string getExpectedOperand2(){
+            return expectedOperand2;
+        }
+        private void setOperation(string value){
+            operation=value;
+        }
+        public string getOperation(){
+            return operation;
+        }
+        private void setResult(string value){
+            result=value;
+        }
+        public string getResult(){
+            return result;
+        }
+        public string getExpectedResult(){
+            return expectedResult;
+        }
+        private void setExpectedValues(){
+            if(operation=="*"){
+                //expectedOperand1
+                if(!operand1.Contains('?'))
+                expectedOperand1=operand1;
+                else{
+                    var operand2Int=Convert.ToInt32(operand2);
+                    var resultInt=Convert.ToInt32(result);
+                    if(operand2Int==0)
+                    expectedOperand1="infinite";
+                    else if(resultInt%operand2Int!=0)
+                    expectedOperand1="unexpected";
+                    else{
+                        expectedOperand1=(resultInt/operand2Int).ToString();
+                    }
+                }
+                //expectedoperand2
+                if(!operand2.Contains('?'))
+                expectedOperand2=operand2;
+                else{
+                    var operand1Int=Convert.ToInt32(operand1);
+                    var resultInt=Convert.ToInt32(result);
+                    if(operand1Int==0)
+                    expectedOperand2="infinite";
+                    else if(resultInt%operand1Int!=0)
+                    expectedOperand2="unexpected";
+                    else{
+                        expectedOperand2=(resultInt/operand1Int).ToString();
+                    }
+                }
+                //expected result
+                if(!result.Contains('?'))
+                expectedResult=result;
+                else{
+                    var operand1Int=Convert.ToInt32(operand1);
+                    var operand2Int=Convert.ToInt32(operand2);
+                    expectedResult=(operand1Int*operand2Int).ToString();
+                }
+            }
+        }
+        public Equation(string operand1,string operand2,string operation,string result){
+            setOperand1(operand1);
+            setOperand2(operand2);

```

Comment:
you can simply say
Operand1 = operand1. It wil set that value to object member. 
Use provided language constructs